final/src/main/java/ua/epam/spring/hometask/converter/impl/AirDateConverterImpl.java

```diff
@@ -0,0 +1,42 @@
+package ua.epam.spring.hometask.converter.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import ua.epam.spring.hometask.converter.AirDateConverter;
+import ua.epam.spring.hometask.converter.AuditoriumConverter;
+import ua.epam.spring.hometask.domain.AirDate;
+import ua.epam.spring.hometask.dto.AirDateDTO;
+
+@Component
```

Comment:
You can add bean name for component to have possibility to work against Converter interface:
@Component("airDateConverter")